Documentation/Kernel/GraphicsSubsystem.md

```diff
@@ -198,17 +198,17 @@ By default, we try to fully-initialize the graphics subsystem, which means we it
 over all PCI devices, searching for VGA compatible devices or Display Controller devices.
 
 We currently natively support QEMU std-vga (and bochs-display) device, VirtIO GPU, VMWare SVGA II adapter,
-and Intel Graphics (Gen 4 only). We try our best to avoid using a pre-initialized framebuffer, so
+and Intel GPU (Gen 4 only). We try our best to avoid using a pre-initialized framebuffer, so
```

Comment:
Should also stay as Intel Graphics

Reply:
This is intentional, because future generations are not merely about graphics (2D) but it's an entire processor unit.